packages/shared/src/components/cards/WelcomePostCardFooter.tsx

```diff
@@ -21,12 +24,20 @@ export const WelcomePostCardFooter = ({
       />
     );
   }
-
   if (post.content) {
     return (
-      <p className="px-2 break-words line-clamp-6 typo-callout">
-        {post.content}
-      </p>
+      <>
+        <p className="px-2 break-words line-clamp-6 typo-callout">
+          {contentRef?.current?.textContent}
+        </p>
+        <div
+          ref={contentRef}
+          className="hidden"
+          dangerouslySetInnerHTML={{
+            __html: sanitize(post.contentHtml, { ADD_ATTR: ['target'] }),
```

Comment:
I would memoize this call, I think it does a lot of things so could be expensive when rendering a lot of cards.

Reply:
Good point. Was also worried about the performance difference after this. Memoizing is the way to go.